docs/update_and_migration/from_4.5/update_from_4.5.md

```diff
@@ -0,0 +1,275 @@
+        PRIMARY KEY(hash)
+    ); 
+    COMMENT ON COLUMN ibexa_payment_token.details IS '(DC2Type:object)';
+    ```
+
+## Run data migration
+
+### Dashboard [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+
+If you are using [[= product_name_exp =]] or [[= product_name_com =]],
+you must run data migration required by the dashboard and other features to finish the upgrade process:
+
+```bash
+php bin/console ibexa:migrations:import vendor/ibexa/dashboard/src/bundle/Resources/migrations/structure.yaml --name=2023_09_23_14_15_dashboard_structure.yaml
+php bin/console ibexa:migrations:import vendor/ibexa/dashboard/src/bundle/Resources/migrations/permissions.yaml --name=2023_10_10_16_14_dashboard_permissions.yaml
+php bin/console ibexa:migrations:migrate --file=2023_09_23_14_15_dashboard_structure.yaml --file=2023_10_10_16_14_dashboard_permissions.yaml
+```
+
+## Revisit mandatory configuration
+
+### Dashboard [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+
+Define "Dashboards" location as contextual tree root:
+
+```yaml
+ibexa:
+    system:
+        # ...
+        admin_group:
+            content_tree_module:
+                contextual_tree_root_location_ids:
+                    #...
+                    - 67 # Dashboards (clean installation)
+```
+
+### User profile
+
+Ibexa DXP v4.6 introduced user profile for Backoffice users, allowing users to upload avatars, and provide personal information.
+
+This feature is optional, and you can disable it by setting `enabled` flag to `false` in `ibexa.system.<scope>.user_profile` configuration:
+
+```yaml
+# /config/packages/ibexa_admin_ui.yaml
+ibexa:
+    system:
+        # ...
+        admin_group:
+            user_profile:
+                enabled: false
+```
+
+To enable the user profile, you must specify Content Type identifiers which represent the "editor" user, and field groups to be rendered in the user profile summary:
+
+```yaml
+# /config/packages/ibexa_admin_ui.yaml
+ibexa:
+    system:
+        # ...
+        admin_group:
+            user_profile:
+                enabled: true
+                content_types: ['editor']
+                field_groups: ['about', 'contact']
+```
+
+You can use your own Content Type that represents the Back Office user, or use the default one provided by [[= product_name =]]:
+
+```bash
+php bin/console ibexa:migrations:import vendor/ibexa/installer/src/bundle/Resources/install/migrations/2023_12_07_20_23_editor_content_type.yaml --name=2023_12_07_20_23_editor_content_type.yaml
+php bin/console ibexa:migrations:import vendor/ibexa/installer/src/bundle/Resources/install/migrations/2024_01_09_22_23_editor_permissions.yaml --name=2024_01_09_22_23_editor_permissions.yaml
+php bin/console ibexa:migrations:migrate --file=2023_12_07_20_23_editor_content_type.yaml --file=2024_01_09_22_23_editor_permissions.yaml
+```
+
+More information ...
```

Comment:
I think we can safely remove this line

Reply:
Removed in d4f5f96977b114b4c97a9381dea12f99c3128a18